src/controllers/media.ts

```diff
@@ -39,57 +90,96 @@ export const getByOsdbHash = async(ctx: Context) => {
     throw err;
   }
 
+  // Fail early if OpenSubtitles reports that it did not recognize the hash
   if (!osMeta.metadata) {
     await FailedLookups.updateOne({ osdbHash }, {}, { upsert: true, setDefaultsOnInsert: true });
     return ctx.body = MESSAGES.notFound;
   }
 
-  const newMetadata = {
-    title: osMeta.metadata.title,
+  let newMetadata = {
+    actors: _.isEmpty(Object.values(osMeta.metadata.cast)) ? null : Object.values(osMeta.metadata.cast),
```

Comment:
Not required, but we could use `_.values` which I find more elegant & ever so slightly faster